src/main/java/com/gluonhq/substrate/target/AndroidTargetConfiguration.java

```diff
@@ -268,6 +278,26 @@ String getLinkOutputName() {
         return linkFlags;
     }
 
+    @Override
+    Predicate<Path> getTargetSpecificNativeLibsFilter() {
+        return this::checkFileArchitecture;
+    }
+
+    private boolean checkFileArchitecture(Path path) {
+        try {
+            ProcessRunner pr = new ProcessRunner(objdump.toString(), "-f", path.toString());
+            pr.showSevereMessage(false);
+            int op = pr.runProcess("objdump");
+            if (op == 0) {
+                return pr.getResponses().stream().anyMatch(line -> line.contains("architecture: " + Constants.ARCH_AARCH64));
```

Comment:
better to replace with the getArch() on the targetTriplet (e.g. when we support x86-64 emulator/devices)

Reply:
ok